### DISCUSSION TITLE
Question about LTSP5

### DISCUSSION THREAD

**Discussion by tichzvidzayi** [ltsp/ltsp#236] [Q&A]
LTSP5 thin clients using more bandwidth (90%) , I am currently experimenting with 1 thin client ( Mecer Xtreme Jupiter Mini PC  4GB RAM) on 2 NIC one connected to Internet and LAN , I have tried to use both 100Mbps and I Gigabit  switch but the results is still the same the server sends around 90% of the available connection

**Comment by alkisg** [comment]
Hi tzvidzie,

you put the question in the title instead of the body; I edited it for you:
[https://github.com/ltsp/community/issues/90](https://github.com/ltsp/community/issues/90)

LTSP5 questions go to the ltsp-discuss mailing list: 
https://sourceforge.net/p/ltsp/mailman/

Here in github we only discuss questions about the new LTSP, which currently is version 20.01. To install the new LTSP, see https://ltsp.org.
